apps/src/templates/Congrats.jsx

```diff
@@ -8,6 +8,8 @@ import TeachersBeyondHoc from './TeachersBeyondHoc';
 
 const styles = {
   container: {
+    width: '100%',
+    maxWidth: 970,
```

Comment:
this should import: https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/style-constants.scss#L14
Right now will this break some css on the page as discussed, since our default is 940, which is smaller than 970?

Reply:
Good catch, will use the import!

This page gets rendered at full width (no `.container` div).  I think because of this line?  Maybe we can use the same fix for the homepage.  https://github.com/code-dot-org/code-dot-org/blob/c4c807ca221a79702ed374258b902cf9935d2009/dashboard/app/controllers/congrats_controller.rb#L3

Reply:
👍 cool I'll look at that